### ISSUE TITLE
Add sudo password to autoscaled machines

### ISSUE THREAD

**Issue by mrnicegyu11** [ITISFoundation/osparc-ops-environments#673]
Follows up on https://github.com/ITISFoundation/osparc-ops-environments/issues/559

```[tasklist]
### Tasks
- [ ] https://github.com/ITISFoundation/osparc-ops-environments/issues/856
- [x] Update monitoring scripts to use password when needed @sanderegg
- [x] [Autoscaling set sudo password on EC2 boot](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1167)
- [x] Generate new AMI (with sudo password in boot script) (https://git.speag.com/oSparc/osparc-infra/-/merge_requests/268)
- [x] ~~[Optional] Harden security (e.g. boot script permissions and location)~~ won't do @YuryHrytsuk @sanderegg
- [x] Deploy to master (https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1173)
- [x] Deploy to STAG (https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1174)
- [x] Deploy to PROD (https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/1175)
- [x] Clean up old (unused) AMIs @sanderegg
```

**Comment by YuryHrytsuk** [comment]
@mrnicegyu11 I think it is fixed?

**Comment by mrnicegyu11** [comment]
Not quite, but thanks for the reminder, it is in fact now *unblocked*.

Dear @sanderegg , let's maybe have a quick chat or talk if and in which code it would impact you if we set a sudo-password on the autoscaled machines (already baked into the AMI with ansible). Please let me know, not urgent ;) Thanks!

**Comment by sanderegg** [comment]
> Not quite, but thanks for the reminder, it is in fact now _unblocked_.
> 
> Dear @sanderegg , let's maybe have a quick chat or talk if and in which code it would impact you if we set a sudo-password on the autoscaled machines (already baked into the AMI with ansible). Please let me know, not urgent ;) Thanks!

@mrnicegyu11 nothing against it.

**Comment by YuryHrytsuk** [comment]
Done in https://git.speag.com/oSparc/osparc-infra/-/merge_requests/255. As soon as it is merged, this issue can be closed

**Comment by YuryHrytsuk** [comment]
> Done in https://git.speag.com/oSparc/osparc-infra/-/merge_requests/255. As soon as it is merged, this issue can be closed

This does not work. It requires further investigation
